sites/svelte-5-preview/src/routes/docs/content/01-api/02-runes.md

```diff
@@ -490,14 +490,24 @@ let { a, b, c, ...everythingElse }: MyProps = $props();
 >
 > ...TypeScript [widens the type](https://www.typescriptlang.org/play?#code/CYUwxgNghgTiAEAzArgOzAFwJYHtXwBIAHGHIgZwB4AVeAXnilQE8A+ACgEoAueagbgBQgiCAzwA3vAAe9eABYATPAC+c4qQqUp03uQwwsqAOaqOnIfCsB6a-AB6AfiA) of `x` to be `string | number`, instead of erroring.
 
-Props cannot be mutated, unless the parent component uses `bind:`. During development, attempts to mutate props will result in an error.
+By default props are treated as readonly, meaning reassignments will not propagate upwards and mutations will result in a warning. You will also get an error trying to `bind:` to them. To declare props as bindable, use [`$bindable()`](#bindable).
```

Comment:
Are reassignments also a warning, or do they just silently not work? It's also not clear to me here that these are dev-time-only warning (which I assume they are) - and I'm not clear on which things will result in a warning versus result in an error, and whether that's a runtime error/warning (I assume so) or a compile-time error/warning (I assume not, unless you're relying on types).

Reply:
Let me know if the refined sentences are more clear.